.github/workflows/ci.yml

```diff
@@ -36,3 +36,35 @@ jobs:
 
       - name: Run tests
         run: npm test
+
+  publish:
```

Comment:
publish-to-docker

Reply:
Or can this be in its own file?

Reply:
It's better to have this in the same file and just use various conditionals to trigger this so that we make sure we're only building this when other CI passes.

Reply:
Fixed the name as requested.